src/js/main.js

```diff
@@ -140,6 +151,14 @@ import '../css/list.css';
         refreshEvents(filteredEvents, 1);
     };
 
+    var clearFields = function() {
+      $('#q').val('');
+      $('#search-wrapper').find('select#location-dropdown')[0].value = '';
```

Comment:
Just noting an inconsistency here...at Ln. 120 the code is picking up the Location & Audience dropdown elements directly from the id. Down here, the code looks for the element as a child of `#search-wrapper`. Any reason for the variation?

Reply:
Right! I had a bit of a back and forth trying to figure out if `.find()` would be more performant than what is essentially `getElementById` since `.find()` was used in this codebase before. From what I see online, the `.find()` would be more useful for something like line 156 where we set a value, but we can use the id selector above. I may be wrong on that, can fix to .find() throughout.